modules/apps/commerce/commerce-order-content-web/src/main/resources/META-INF/resources/js/util/index.js

```diff
@@ -14,6 +14,7 @@ const EDITABLE_FIELDS = [
 	'shippingAddress',
 	'shippingMethod',
 ];
+export const PAYMENT_METHOD_TYPE_OFFLINE = 2;
```

Comment:
what if somebody defines his own offline payment method?

Reply:
If the method type is offline, they don't have to go through the payment process and the order can be checked out immediately

Reply:
so by saying the type is offline it automatically gets this number set?

Reply:
Yes, this is the value for the offline payment methods type